src/config.cpp

```diff
@@ -86,12 +128,39 @@ static inline void print_version(std::ostream& os)
 static inline void print_usage(std::ostream& os, const char* name,
                                const po::options_description& desc)
 {
+    static constexpr char argument_detail[] =
+        R"(
+Arguments to options:
+    PATH        A filesystem path.  It can be either relative to the
```

Comment:
Please include the changes of ae40d227de88d221d3151bf63ed218c959d04f4f on master